src/Middleware/src/Headstart.API/Commands/OrderCommand.cs

```diff
@@ -31,20 +37,74 @@ public class OrderCommand : IOrderCommand
         private readonly ILocationPermissionCommand _locationPermissionCommand;
         private readonly IPromotionCommand _promotionCommand;
         private readonly IRMACommand _rmaCommand;
+        private readonly AppSettings _settings;
+        private readonly ISendgridService _sendgridService;
 
         public OrderCommand(
             ILocationPermissionCommand locationPermissionCommand,
             IOrderCloudClient oc,
             IPromotionCommand promotionCommand,
-            IRMACommand rmaCommand
+            IRMACommand rmaCommand,
+            AppSettings settings,
+            ISendgridService sendgridService
             )
         {
 			_oc = oc;
             _locationPermissionCommand = locationPermissionCommand;
             _promotionCommand = promotionCommand;
             _rmaCommand = rmaCommand;
+            _settings = settings;
+            _sendgridService = sendgridService;
 		}
 
+        public async Task<HSLineItem> OverrideQuotePrice(string orderID, string lineItemID, decimal quotePrice)
+        {
+            var token = await GetTokenForQuoteOrders();
+            var linePatch = new PartialLineItem { UnitPrice = quotePrice };
+            var updatedLineItem = await _oc.LineItems.PatchAsync<HSLineItem>(OrderDirection.All, orderID, lineItemID, linePatch, accessToken: token);
+            var orderPatch = new PartialOrder { xp = new { QuoteStatus = QuoteStatus.NeedsBuyerReview } };
+            var updatedOrder = await _oc.Orders.PatchAsync<HSOrder>(OrderDirection.All, orderID, orderPatch, accessToken: token);
+            // SEND EMAIL NOTIFICATION TO BUYER
+            await _sendgridService.SendQuotePriceConfirmationEmail(updatedOrder, updatedLineItem, updatedOrder.xp?.QuoteBuyerContactEmail);
+            return updatedLineItem;
+        }
+
+        public async Task<ListPage<HSOrder>> ListQuoteOrders(MeUser me, QuoteStatus quoteStatus)
+        {
+            var supplierID = me.Supplier?.ID;
+            var token = await GetTokenForQuoteOrders();
+            var filters = new Dictionary<string, object>
+            {
+                ["xp.QuoteSupplierID"] = supplierID != null ? supplierID : "*",
+                ["IsSubmitted"] = false,
+                ["xp.OrderType"] = OrderType.Quote,
+                ["xp.QuoteStatus"] = quoteStatus
+            };
+            // For now, assuming no more than 100 quote orders. TO-DO - allow for scrolling to retrieve additional pages.
+            var quoteOrders = await _oc.Orders.ListAsync<HSOrder>(OrderDirection.Incoming, pageSize: 100, filters: filters, accessToken: token);
+            return quoteOrders;
+        }
```

Comment:
If scrolling isn't possible in this PR can you use the ListAll helper to get all orders? It could be a pretty bad bug if a supplier isn't able to see a quote order